foundry/applications/formApplications/folderConfig.d.ts

```diff
@@ -1,6 +1,52 @@
 /**
  * Edit a folder, configuring its name and appearance
  */
-declare class FolderConfig extends FormApplication {
-  _updateObject(event?: Event, formData?: object): Promise<any>; // TODO: Add correct return type
+declare class FolderConfig extends FormApplication<FolderConfig.Data, Folder> {
+  /**
+   * @override
+   * @defaultValue
+   * ```typescript
+   * mergeObject(super.defaultOptions, {
+   *   classes: ["sheet", "folder-edit"],
+   *   template: "templates/sidebar/folder-edit.html",
+   *   width: 360
+   * });
+   * ```
+   */
+  static get defaultOptions(): FormApplication.Options;
+
+  /**
+   * @override
+   */
+  get id(): string;
+
+  /**
+   * @override
+   */
+  get title(): string;
+
+  /**
+   * @param options - (unused)
+   * @override
+   */
+  getData(options?: Application.RenderOptions): Promise<FolderConfig.Data>;
+
+  /**
+   * @param event - (unused)
+   * @override
+   */
+  protected _updateObject(event: Event, formData: FolderConfig.FormData): ReturnType<Folder['update']>;
+}
+
+declare namespace FolderConfig {
+  interface Data {
+    folder: Folder.Data;
+    sortingModes: {
+      a: 'FOLDER.SortAlphabetical';
+      m: 'FOLDER.SortManual';
```

Comment:
Maybe make this extendable?

Reply:
For posterity: These are hard coded in the method.